client/my-sites/plan-features/actions.jsx

```diff
@@ -43,7 +43,7 @@ const PlanFeaturesActions = ( {
 		upgradeButton = (
 			<Button className={ classes } href={ manageHref } disabled={ ! manageHref }>
 				<Gridicon size={ 18 } icon="checkmark" />
-				{ canPurchase ? translate( 'Your plan' ) : translate( 'Current plan' ) }
+				{ canPurchase ? translate( 'Manage your plan' ) : translate( 'View plan' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Manage Plan', { context: 'Link to current plan from /plans/'} )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).